AuctionHouse/Overview.md

```diff
@@ -0,0 +1,101 @@
+# Auction House: Overview
+
+## Introduction
+Auction House is a program that allows users to exchange assets within the Solana blockchain. 
+
+There are plenty of ways to exchange assets on Solana, so why another protocol focused on solving this problem? Let's dive deep into that.
```

Comment:
We can call programs any things (protocol, smart contract, etc.) but let's keep a consistent language here and stick with "program" everywhere.

```suggestion
There are plenty of ways to exchange assets on Solana, so why another program focused on solving this problem? Let's dive deep into that.
```